### ISSUE TITLE
Detect and show red plague portals

### ISSUE THREAD

**Issue by Eccenux** [IITC-CE/ingress-intel-total-conversion#598]
In details index 16 shows `unknown` faction (owner). This for guid `982cdfea9f994dcdb7b08e9c17d6a517.16`:
```
{"result":["p","N",54533043,18454076,1,25,8,"http://lh3.googleusercontent.com/teKTRb31AQxjYS2dDeD6UohtgTYM1wACpHPHmii7Cy6PbAjPrfw5yoXLRLPVsQbD5z_QV209Hsa4rjYtpF2euWvRNPs","Kapliczka przy ko\u015bciele Matki Boskiej R\u00f3\u017ca\u0144cowej",[],false,false,null,1668324131639,[null,null,null,null],[["unknown",1,250],["unknown",1,250],["unknown",1,250],["unknown",1,250],["unknown",1,250],["unknown",1,250],["unknown",1,250],["unknown",1,250]],"unknown",["","",[]],3]}
```
Also all resonators are correctly shown as unknown. Might use a red color for both. For good contrast `#f47b92` should be OK (4.5 contrast).
![obraz](https://user-images.githubusercontent.com/1045235/201521742-fb07dcec-50a0-4ba4-99ef-d7a4dd98adf7.png)

In getEntities request you can detect red plague by resonators.  See the last on this short list:
```
					["269b94d4bd234da7844bbef57a888726.16", 1663151678415, ["p", "N", 54533198, 18453378, 1, 0, 0, "http://lh3.googleusercontent.com/sc1fOxXslV410V0GmSTTO7C9YZxrXSg1GJncivycw7nz3MnELDazy1nhBqhivF1nLdjaSkeQ_OWRkzByhu-Is7_ja24H", "Ko\u015bci\u00f3\u0142 Matki Boskiej R\u00f3\u017ca\u0144cowej w Gdyni", ["sc5_p"], false, false, null, 1663151678415, null, null, null, null, 3]],
					["f28c1be0f0e94366b68b1b4568c2b5c9.16", 1668339481965, ["p", "R", 54532364, 18454170, 6, 70, 8, "http://lh3.googleusercontent.com/1sSVybxwPyH5BSBcHjDhOKIyqkmVrfMFadeljHOooIXbFccGQztlxvtQLdogoN67OYnEbmfXZTigWZp0YPOucXi8EBA", "Mural Parkour", [], false, false, null, 1668339481965]],
					["982cdfea9f994dcdb7b08e9c17d6a517.16", 1668324131639, ["p", "N", 54533043, 18454076, 1, 25, 8, "http://lh3.googleusercontent.com/teKTRb31AQxjYS2dDeD6UohtgTYM1wACpHPHmii7Cy6PbAjPrfw5yoXLRLPVsQbD5z_QV209Hsa4rjYtpF2euWvRNPs", "Kapliczka przy ko\u015bciele Matki Boskiej R\u00f3\u017ca\u0144cowej", [], false, false, null, 1668324131639, null, null, null, null, 3]],
```
If I'm reading this correctly, then the portal is neutral ("N"), but also has 8 resonators with 25%. So should easily detectable on the map.
If the portal is N and res > 0 ⇒ red plague portal.

**Comment by Eccenux** [comment]
To avoid problem with highlighters I think it would be good to use different shape. Probably easiest would be to use a triangle... But that would be kind of boring.

So maybe:
https://commons.wikimedia.org/wiki/File:Virus_malware_hazard_icon.svg
Or more like in intel-x, but a simpler shape:
https://commons.wikimedia.org/wiki/File:Corona_virus_icon.svg

**Comment by xscreach** [comment]
Fixed In last release - v0.35.0

**Comment by Suburbanno** [comment]
close
done https://github.com/IITC-CE/ingress-intel-total-conversion/pull/597
